homework/App.js

```diff
@@ -11,21 +11,40 @@ class App {
    * Initialization
    * @param {string} url The GitHub URL for obtaining the organization's repositories.
    */
-  async initialize(url) {
-    // Add code here to initialize your app
-    // 1. Create the fixed HTML elements of your page
-    // 2. Make an initial XMLHttpRequest using Util.fetchJSON() to populate your <select> element
 
+  async initialize(url) {
     const root = document.getElementById('root');
-
-    Util.createAndAppend('h1', root, { text: 'It works!' }); // TODO: replace with your own code
-
+    const pre = Util.createAndAppend('pre', root, { id: 'pre', class: 'pre' });
+    Util.createAndAppend('span', pre, { text: 'HYF Repositories', class: 'logoName' });
+    Util.createAndAppend('div', root, { id: 'container', class: 'bodyInformation' });
     try {
+      // Fetch all repository's
       const repos = await Util.fetchJSON(url);
-      this.repos = repos.map(repo => new Repository(repo));
-      // TODO: add your own code here
+      repos.sort((x, y) => x.name.localeCompare(y.name));
+
+      // Create instance's for each repository
+      this.repos = repos.map(repo => new Repository(repo)); // to render Repository
+
+      // Start value for each repository inside HTML selector
+      let value = 0;
+
+      // Create and render the Selector
+      const select = Util.createAndAppend('select', pre, { id: 'select', class: 'select' });
+      this.repos.forEach(repo =>
+        Util.createAndAppend('option', select, { text: repo.name(), value: value++ }),
```

Comment:
I don't think you need a new variable `value` for the select options. `forEach` callback function has three arguments(element, index and the original array). You can make use `index` instead of `value` here then 🙂

Reply:
Yes you are right. when i was writhing this code i tried to use the index but it didn't work, then i switched to try using any simple way to skip my issues and look to another. I have forgot to go back to fix it. 
But now thanks your comments, i would try it out and fix it 🙂.